src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7589,6 +7589,8 @@ public sealed partial class StringBuilder : System.Runtime.Serialization.ISerial
         public System.Text.StringBuilder Append(long value) { throw null; }
         public System.Text.StringBuilder Append(object value) { throw null; }
         public System.Text.StringBuilder Append(System.ReadOnlySpan<char> value) { throw null; }
+
```

Comment:
Nit, please remove this blank line, the file is periodically (sort of) auto generated and it will be a diff.

Reply:
Done